main/chainlink-integration.md

```diff
@@ -0,0 +1,68 @@
+# Chainlink Integration
+
+This document explains how you can consume [Chainlink
+oracles](https://chain.link) when unit testing, integration testing, and
+actually on-chain.
+
+## Overview
+
+Using Chainlink on Agoric provides two main features:
+- price feeds exposed on Agoric via the on-chain `home.priceAuthority`, an
+  officially-sponsored [price authority](/zoe/guide/price-authority.md) built
+  from aggregating several Chainlink nodes.
+- [Chainlink's Any API](https://docs.chain.link/docs/request-and-receive-data)
+  to initiate a job on a single oracle and return its results
```

Comment:
This doesn't sync with the intro to it, which is "Using Chainlink provides two main features:".  Maybe something like change "to initiate" to "can initiiate"